DuckDuckGo/SubscriptionDebugViewController.swift

```diff
@@ -125,22 +123,22 @@ final class SubscriptionDebugViewController: UITableViewController {
             }
         
         case .environment:
-            let staging = SubscriptionPurchaseEnvironment.ServiceEnvironment.staging
-            let prod = SubscriptionPurchaseEnvironment.ServiceEnvironment.production
+            let staging = SubscriptionEnvironment.ServiceEnvironment.staging
+            let prod = SubscriptionEnvironment.ServiceEnvironment.production
+            let currentEnv = subscriptionManager.currentEnvironment.serviceEnvironment
             switch EnvironmentRows(rawValue: indexPath.row) {
             case .staging:
                 cell.textLabel?.text = "Staging"
-                cell.accessoryType = SubscriptionPurchaseEnvironment.currentServiceEnvironment == staging ? .checkmark : .none
+                cell.accessoryType = currentEnv == staging ? .checkmark : .none
```

Comment:
No need to define `staging` and `prod` you can just use type inference `currentEnv == .staging`

Reply:
leftover removed